### DISCUSSION TITLE
希望开发typecho评论转移至halo插件

### DISCUSSION THREAD

**Discussion by tuantuanlive** [halo-dev/halo#6069] [Ideas]
### 你当前使用的版本

typecho1.2.1

### 描述一下此特性

当前halo应用市场只有typecho文章转入halo,并没有评论转移,通过第三方评论插件,例如Twikoo是能实现typecho评论转移,但始终是在第三方系统上,需要额外去搭建应用,想将评论集成halo当中!

**Comment by ShiinaKin** [comment]
请到插件仓库提这个issue

**Comment by ruibaby** [comment]
我将转移到 Discussion 的 Ideas 分类，期望社区有小伙伴有兴趣开发此插件。
